bagitobjecttransfer/bagitobjecttransfer/settings/base.py

```diff
@@ -162,3 +165,60 @@
 CLAMAV_ENABLED = config('CLAMAV_ENABLED', cast=bool, default=True)
 CLAMAV_HOST = config('CLAMAV_HOST', default='clamav')
 CLAMAV_PORT = config('CLAMAV_PORT', cast=int, default=3310)
+
+# Pipeline configuration
+
+STATICFILES_STORAGE = 'pipeline.storage.PipelineManifestStorage'
+
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'pipeline.finders.PipelineFinder',
+)
+
+# create separate minified stylesheets and javascript files for each app
+PIPELINE = {
+    'YUGLIFY_BINARY': os.path.join(BASE_DIR, 'node_modules/.bin/yuglify'),
+    'STYLESHEETS': {
+        'caais_styles': {
+            'source_filenames': (
+                'caais/css/*.css',
+            ),
+            'output_filename': 'caais/css/min.css',
+            'extra_context': {
+                'media': 'screen,projection',
+            },
+        },
+        'recordtransfer_styles': {
+            'source_filenames': (
+                'recordtransfer/css/*.css',
+            ),
+            'output_filename': 'recordtransfer/css/min.css',
+            'extra_context': {
+                'media': 'screen,projection',
+            },
+        },
```

Comment:
Just tested the app out after reviewing and realized one thing that needs to be changed.

There needs to be another group here, called `recordtransfer_submission_detail_styles`.

The stylesheet `submission_detail.css` is specific to the `submission_detail.html` template, and has styles that conflict with the rest of the application.

Can you make another group here for just that one file? And then that file should be loaded in the submission_detail.html template.

Reply:
Addressed by:
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/186/commits/af637d8e33d3b97fde7fd22a9159be71cce55583
[moved submission detail style for recordtransfer into separate folder](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/186/commits/35269487ccf39128d6c17a261828db9c1ee94e96)

Reply:
Did some code organisation too, to keep static asset naming and organisation consistent